File: PSET1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    int height;
    do
    {
        height = get_int("Height: ");
    } while (height > 8 || height <1);

    for (int row = 0; row < height; row++)
    {
        for (int space = height -1; space >= row+1; space--)
            printf(" ");

        for (int hash = 1; hash <= row+1; hash++)
            printf("#");

        printf("\n");

    }
}

File: PSET6/mario.py
from cs50 import get_int

def main():
    height = get_int("Height: ")
    if height >= 1 and height <=8:
        for i in range(height):
            print(" " * (height-i-1) + "#" * (i+1) + "  " + "#" * (i+1))

    else:
        main()


main()

